staking/programs/publisher-caps/src/lib.rs

```diff
@@ -71,22 +146,89 @@ impl Ord for PublisherCap {
 
 #[account(zero_copy)]
 pub struct PublisherCaps {
-    pub timestamp:      i64,
-    pub num_publishers: u64,
-    pub caps:           [PublisherCap; MAX_CAPS],
+    pub write_authority:               Pubkey,
+    pub is_verified:                   u8,
```

Comment:
can't we use boolean?

Reply:
No boolean is not Pod because a Pod type should allow any byte pattern
https://docs.rs/bytemuck/latest/bytemuck/trait.Pod.html